BEACON-V2-Model/biosamples/requestParameters.json

```diff
@@ -3,12 +3,18 @@
   "biosample": {
     "type": "object",
     "properties": {
-      "id": {
-        "type": "string"
+      "ids": {
+        "type": "array",
+        "items": {
+          "type": "string"
+        },
+      },
+      "individualIds": {
```

Comment:
I'm not sure this is required here. If you want to make a query which specifies individualIDs you'd send the individuals model as part of the query and specify the IDs within there.